taqueria-toolkit/TaqError.ts

```diff
@@ -0,0 +1,5 @@
+export class TaqError extends Error {
+	public isTaqError = true;
+}
+export const isTaqError = (err: unknown): err is TaqError =>
+	typeof err === 'object' && (err as object).hasOwnProperty('isTaqError');
```

Comment:
I would return "isSubclassOf(TaqError)" and you can elide the isTaqError member.
Small fish.

Reply:
Hmm, is that new? I'm not aware of TypeScript having this keyword or built-in function. I'd be interested to see how that gets transpiled to JS. We should probably make point of bumping our TS versions as right now we're using 4.7.3.

Reply:
Right, when I said "whatever the equivalent is in TS" I didn't realize the type system has no support for this construct. So never mind :grin: